MDGameSynchronizer/MDGameSynchronizer.cs

```diff
@@ -524,16 +530,11 @@ private void OnPingTimerTimeout(Timer timer, int PeerId)
             }
 
             // Send ping request
-            if (MDStatics.IsClient() || GameClock == null)
-            {
-                RpcId(PeerId, nameof(RequestPing), OS.GetTicksMsec());
-            }
-            else
-            {
-                uint ping = (uint) GetPlayerPing(PeerId);
-                uint estimate = GetPlayerTicksMsec(PeerId) + ping;
-                RpcId(PeerId, nameof(RequestPing), OS.GetTicksMsec(), estimate, GameClock.GetTickAtTimeOffset(ping));
-            }
+            uint ping = (uint) GetPlayerPing(PeerId);
```

Comment:
The old code was like this,

```csharp
        // Send ping request
        if (MDStatics.IsClient() || GameClock == null)
        {
            RpcId(PeerId, nameof(RequestPing), OS.GetTicksMsec());
        }
        else
        {
            uint ping = (uint)GetPlayerPing(PeerId);
            uint estimate = GetPlayerTicksMsec(PeerId) + ping;
            RpcId(PeerId, nameof(RequestPing), OS.GetTicksMsec(), estimate, GameClock.GetTickAtTimeOffset(ping));
        }
```

This IF still needs to be there for cases where GameClock is not enabled. So please add the IF statement back with the alternative ping method.